packages/x-bundle/src/XBundle.ts

```diff
@@ -37,6 +39,22 @@ export class XBundle extends Bundle<IXBundleConfig> {
     live: {
       debug: false,
     },
+    cacheConfig: {
+      store: "memory",
+      storeConfig: {
+        memory: 100,
+        ttl: 60,
+      },
+      ttl: 60 /*seconds*/,
+      isCacheableValue: () => true,
+      refreshThreshold: 1,
+      userBoundness: true,
```

Comment:
shouldn't userBoundness be configured at resolver level? I want some queries to be user bounds, others not, or are these default values? If so, pls use "resolverDefaults: {}"